### DISCUSSION TITLE
专业版收不到验证码

### DISCUSSION THREAD

**Discussion by lzlore** [getlantern/lantern-forum-cn#406] [General]

**Comment by lzlore** [comment]
@Mike777Khoo 请帮忙看看怎么处理

**Reply by MaxMKMax** [reply]
@lzlore @wennuanbeijiguang 
您好. 请问您可以提供我邮箱和购买单据吗? 谢谢! 我来帮你们提交给负责人来拿激活码. 两天后请留意邮箱和GitHub.

**Reply by lzlore** [reply]
1

**Comment by wennuanbeijiguang** [comment]
@Mike777Khoo 请问麦克 专业版的恢复 还需要等待多久能解决？

**Comment by MaxMKMax** [comment]
@wennuanbeijiguang 您好. 我给您发激活码了. 邮件标题: 蓝灯激活码 from mike@getlantern.org

用激活码就能登陆专业版了. 不好意思让您久等了. 谢谢! 

还有什么需要可以参考这里:
https://lantern.io/zh-Hans/faq

**Comment by TTLYZ04** [comment]
@Mike777Khoo  遇到同样问题，无法登录专业版本，一直收不到邮件，请帮忙处理，谢谢。 注册邮箱是  --, 支付凭证：

![Uploading payment.PNG…]()

**Reply by MaxMKMax** [reply]
@TTLYZ04 好的. 我已经帮您提交问题上去处理了.

**Reply by chenglei841102** [reply]
> @TTLYZ04好的。我已经帮您提交问题了。

你好！33841927@qq.com 我重新安装了新版本的蓝灯现在这个邮箱收不到验证码 无法使用专业版

**Comment by Join-7** [comment]
@Mike777Khoo 您好，已购买专业版但未收到激活码。注册邮箱是--。支付凭证

**Reply by MaxMKMax** [reply]
@Join-7 您好. 好的, 已经帮您提交问题上去. 请您过几天再回来查看GitHub或邮箱. 谢谢您支持蓝灯!

**Comment by courage-LEO** [comment]
您好，我也是购买了lantern专业版，但是在手机ios系统上没有找到授权吗，邮箱是: --

**Reply by MaxMKMax** [reply]
@courage-LEO 您好. iOS暂时没有授权码功能. 技术团队在努力弄iOS新版本了! 我们之后会帮您延长专业版当补偿. 非常抱歉. 谢谢您!

**Comment by courage-LEO** [comment]
感谢您的回复。

期待团队的尽快尽快更新，另外我想确认目前ios的lantern是否是免费不稳定状态的？

Mike K ***@***.***> 于2024年4月26日周五 13:14写道：

**Reply by MaxMKMax** [reply]
@courage-LEO 您好. 不客气的. 暂时是不稳定的. 因为iOS很久没有版本更新了. 但以更新好, 就是可以恢复状态了.

您那里是遇到什么问题吗? 帮我提交个报告然后给我那个 8 位数票号. 我来反馈.

**Comment by courage-LEO** [comment]
您好，


现在没什么了，就是之前的问题，我在电脑上买的一年的，但是ios上没有办法授权。


关于票号是指的什么，是购买记录嘛？提交报告的话，我手机上的lantern无法提交任何问题……提交就是会出现错误提示（strconv.atoi:parsing"":invalid
syntax)


[image: 431714117092_.pic.jpg]




Mike K ***@***.***> 于2024年4月26日周五 15:19写道：

**Comment by Join-7** [comment]
您好，我申请的退款 请问什么时间能处理，已经不止一周了
发自我的 iPhone

在 2024年4月24日，01:55，Mike K ***@***.***> 写道：

﻿

@Join-7<https://github.com/Join-7> 您好. 好的, 已经帮您提交问题上去. 请您过几天再回来查看GitHub或邮箱. 谢谢您支持蓝灯!

**Comment by Join-7** [comment]
﻿您好，我申请的退款 请问什么时间能处理，已经不止一周了
发自我的 iPhone

在 2024年4月24日，01:55，Mike K ***@***.***> 写道：

﻿

@Join-7<https://github.com/Join-7> 您好. 好的, 已经帮您提交问题上去. 请您过几天再回来查看GitHub或邮箱. 谢谢您支持蓝灯!

**Reply by MaxMKMax** [reply]
@Join-7 您好. 请问您是发邮件到support@getlantern.org申请退款的吗? 我这里只负责看Github , 没有权限负责您提交的邮件的.

**Comment by zhubingge** [comment]
@Mike777Khoo 您好，我之前购买了2年期的专业版本，账号是--；
但是该邮箱一直无法收到授权设备使用专业版本的 6位验证码，麻烦处理一下，很急。
下面是我的支付截图。
麻烦处理一下。
我提交了一个问题报告， lantern返回给我一个问题编码72110062，不知道能不能帮你解决问题。

![微信图片_20240504142617](https://github.com/getlantern/lantern-forum-cn/assets/43376597/b6c4a61f-303e-4cc4-9ef1-2330997c1f85)

**Reply by MaxMKMax** [reply]
@zhubingge  您好. 好的, 刚才已经帮您提交上去了. 周末团队是不上线的, 请您等待星期一晚上. 我会尽快帮您催他们的. 不好意思让您久等了. 谢谢您!

**Reply by zhubingge** [reply]
@Mike777Khoo 大约需要等多久？目前我的邮箱xiaoxinzuopai@sina.com 还是无法接收到验证码，无法授权设备使用专业版。
我希望你能否帮我确认这个账号是否已经是专业版了？因为自己购买后就登录不上去专业版本了。如果是的话，如何能够收到pin验证码？

**Reply by zhubingge** [reply]
> @zhubingge  您好. 好的, 刚才已经帮您提交上去了. 周末团队是不上线的, 请您等待星期一晚上. 我会尽快帮您催他们的. 不好意思让您久等了. 谢谢您!

@Mike777Khoo 您好，我这个问题已经很长时间，快一个月了。至今还没有收到解决的邮件。请问什么时候可以解决呀？急。
我需要，首先确认我的账号已有专业版权限，二是我的邮箱需要接收授权设备的验证码。拜托了。

**Reply by MaxMKMax** [reply]
@zhubingge QJ8HT-VC9GM-YCR8M-RWH2V-PX84J
您好. 这是您专用的验证码. 不好意思让您久等了. 刚刚才受到团队的回复.

**Comment by JerryMouse2022** [comment]
@Mike777Khoo 你好，也是无法收到激活码，邮箱:1328604495@qq.com
![截屏 2024-05-04 15 48 26](https://github.com/getlantern/lantern-forum-cn/assets/101440965/8ee783c6-6e96-48e6-b18d-930b7210ebb2)

**Comment by MaxMKMax** [comment]
@JerryMouse2022  您好. 好的, 刚才已经帮您提交上去了. 周末团队是不上线的, 请您等待星期一晚上. 我会尽快帮您催他们的. 不好意思让您久等了. 谢谢您!

**Comment by MaxMKMax** [comment]
![image](https://github.com/getlantern/lantern-forum-cn/assets/155475142/7acd4329-21df-45a9-b9f8-3ab5c3816e23)

@JerryMouse2022 您好. 付款团队说您的专业版没问题. 直接登陆即可.

**Reply by JerryMouse2022** [reply]
重新email link了下，没问题了

**Reply by MaxMKMax** [reply]
@JerryMouse2022 嗯嗯. 那就好. 谢谢您的答复. 祝您愉快的一天!

**Reply by JerryMouse2022** [reply]
@Mike777Khoo 麻烦再看下我的账户，又不行了，这次通过EMAIL link也不行，邮箱无法收到验证码

**Reply by JerryMouse2022** [reply]
@Mike777Khoo 看了下，是邮件被屏蔽了，能直接给我个验证码吗

**Reply by JerryMouse2022** [reply]
@Mike777Khoo 目前的情况是

> Enter your email and the 25 digit activation code received from the reseller to register your account.

如果用这种方式激活，输入第一次买的时候发给我的25位的专业版激活码，老是说不对，不知道和换邮箱地址有没有关系，

> Authorize Device for Pro

用这种方式来激活，邮箱又无法收到6位的激活码

**Comment by 0GIT-AI** [comment]
@Mike777Khoo您好，我之前购买了2年期的专业版本，账号是[shuojia1823@gmail.com)；
但是该邮箱一直无法收到授权设备使用专业版本的6位验证码，麻烦处理一下，很急。

**Comment by 0GIT-AI** [comment]
@Mike777Khoo您好，我之前购买了2年期的专业版本，账号是[[shuojia1823@gmail.com](mailto:shuojia1823@gmail.com))；
但是该邮箱一直无法收到授权设备使用专业版本的6位验证码，麻烦处理一下，很急。

**Reply by MaxMKMax** [reply]
@0GIT-AI 您好. 好的, 我帮您提交问题处理. 请您也提供我支付单据. 谢谢您!

**Comment by iam189cm** [comment]
![f0869cdf4fe6b8e08496398f0f410d5](https://github.com/user-attachments/assets/e317880d-43be-4865-90df-f002d0ce2d81)
你好，专业版无法激活

**Comment by softlvye123** [comment]
你好无法收到6位的授权码邮件， 刚续费的。 soft_lvye@163.com

**Reply by MaxMKMax** [reply]
@softlvye123 您好, 请您提供我支付单据. 我帮您提交去处理. 谢谢您

**Comment by chen2354** [comment]
@Mike777Khoo 同样问题，购买了收不到验证码，麻烦帮忙看下
chen2354@gmail.com
<img width="634" alt="image" src="https://github.com/user-attachments/assets/b593ed12-79f6-4359-8d31-9d21d64ece13" />

**Comment by chen2354** [comment]
@Mike777Khoo 请问能帮忙看下吗？已经购买好几天了，但是完全不能升级为 pro 账户，速度几乎是不可用的状态。谢谢
chen2354@gmail.com

**Comment by chen2354** [comment]
@Mike777Khoo 您好，我购买了 pro 以后长期无法使用，请帮忙看一下。写了邮件给 customer support，收到的回复是人员不足， ticket 自动关闭了，完全置之不理。如果这里不能解决，麻烦告知能解决问题的客服。如果都不能解决，请退款
<img width="779" alt="image" src="https://github.com/user-attachments/assets/0171952e-178c-4f34-9fb0-cffb0e302f27" />

**Comment by huh824** [comment]
@Mike777Khoo  您好，已购买专业版但未收到激活码。注册邮箱是hu_hai824@126.com(mailto:hu_hai824@126.com),
![20250925143018](https://github.com/user-attachments/assets/b099a423-5d22-4c19-81cf-432e4c1e0b37)

**Comment by suanliao6711-del** [comment]
@Mike777Khoo您好，今天购买了专业版，邮箱未收到连接码，邮箱：suanliao6711@gmail.com，麻烦处理一下，谢谢。
<img width="660" height="1434" alt="IMG_5572" src="https://github.com/user-attachments/assets/50ab0873-3b5e-42f9-b77a-53afd3bc0292" />

**Comment by len531620355-oss** [comment]
@MaxMKMax 您好，我购买的专业版本，账号是[[531620355@qq.com](mailto:531620355@qq.com)]
但是该邮箱一直无法收到授权设备使用专业版本的验证码，麻烦处理一下，很急。

**Reply by len531620355-oss** [reply]
![Uploading 1.jpg…]()

**Comment by PPING1979** [comment]
我的邮箱是pping@189.cn
收不到授权码，请麻烦处理下

**Comment by greySkiesMarsh** [comment]
Hello everyone, 

This is Marsh, the support lead here at Lantern. I'd love to help with any and all account-related issues, but I'm not able to provide direct support in this forum. 

Please submit an issue report from the app, send an email to support@getlantern.org, or through our [portal](https://support.lantern.io) and I will help you as quickly as I can. 

For now, I'm going to lock this discussion, as this forum is not the place for us to provide support for issues like this.
